environments/auth-with-federated/amplify/backend/auth/authwithfederated/authwithfederated-cloudformation-template.yml

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Most of the `environments/*` updates are from `yarn environments pull` using the new CLI beta – `6.5.0-beta.0`